packages/teleterm/src/services/logger/loggerService.ts

```diff
@@ -1,11 +1,38 @@
-import { createLogger as createWinston, format, transports } from 'winston';
+import winston, {
+  createLogger as createWinston,
+  format,
+  transports,
+} from 'winston';
 import { isObject } from 'lodash';
 
 import split2 from 'split2';
 
-import { Logger, NodeLoggerService } from './types';
+import { Logger, LoggerService, NodeLoggerService } from './types';
 
-export default function createLoggerService(opts: Options): NodeLoggerService {
+export function createStdoutLoggerService(): LoggerService {
```

Comment:
Could you add a comment explaining why we need both `createStdoutLoggerService` and `createFileLoggerService`?

Reply:
Done